src/libwalletqt/Wallet.cpp

```diff
@@ -301,7 +308,14 @@ void Wallet::initAsync(
     });
     if (future.first)
     {
-        setConnectionStatus(Wallet::ConnectionStatus_Connecting);
+        if (m_walletImpl->isOffline())
+        {
+            setConnectionStatus(Wallet::ConnectionStatus_Initializing);
```

Comment:
Could you explain this? Why ConnectionStatus_Initializing when offline?

Reply:
When the user opens the wallet in offline mode, it will display "Initializing wallet" on Network status during initAsync, instead of "Connecting", which is the default. It would be contradictory to display a "Connecting" message when the wallet is in offline mode.

Reply:
Could you directly set it to `ConnectionStatus_OfflineMode`?

Reply:
fixed